swift/integration-tests/create_database_utils.py

```diff
@@ -1,15 +1,34 @@
 """
-recreation of internal `create_database_utils.py` to run the tests locally, with minimal
-and swift-specialized functionality
+Simplified version of internal `create_database_utils.py` used to run the tests locally, with
+minimal and swift-specialized functionality
+TODO unify integration testing code across the public and private repositories
 """
 import subprocess
 import pathlib
 import sys
+import shutil
 
 
-def run_codeql_database_create(cmds, lang, keep_trap=True):
+def runSuccessfully(cmd):
+    res = subprocess.run(cmd)
+    if res.returncode:
+        print("FAILED", file=sys.stderr)
```

Comment:
yeah, but by default it also adds level filtering and formatting (for example `ERROR:root:message`). It can be configured, but I don't think it's worth it, especially as this is slated to be thrown away at some point, either reusing the internal test runner or switching to another test framework.